packages/components/src/components/as-switch/as-switch.tsx

```diff
@@ -1,23 +1,24 @@
 import { Component, Event, EventEmitter } from '@stencil/core';
 
 @Component({
+  shadow: false,
```

Comment:
Are all these changes suggested by the linter? 😂

Reply:
yep, vscode linter